src/screens/ConfirmationCode/ConfirmationCodeScreen.js

```diff
@@ -0,0 +1,75 @@
+import React, { useState, useEffect } from 'react';
+import { View, Text } from 'react-native';
+import { Auth } from 'aws-amplify';
+import { CommonActions, useNavigation } from '@react-navigation/native';
+import Input from '../../components/Input/Input';
+import CommonButton from '../../components/CommonButton/CommonButton';
+import * as SCREEN from '../../constants/screen.constant';
+import { toastSuccess, toastError } from '../../utils/toast.utils';
+import styles from './styles';
+
+const ConfirmationCodeScreen = ({ route }) => {
+  const navigation = useNavigation();
+  const email = route.params.email;
+  const [confirmationCode, setConfirmationCode] = useState('');
```

Comment:
Change `confirmationCode` to `confirmation_code`. This is to avoid confusion in naming conventions.